.github/workflows/playwright.yml

```diff
@@ -0,0 +1,43 @@
+name: Playwright Tests
+
+on: push
```

Comment:
do we want to run on pull request too?

Reply:
I saw Duval only had it on push (and even set it to ignore the main branch), and if we run it on push AND pull_request that doubles the number of times the tests run from 3 actions to 6, and I was worried about test flakiness. BUT the tests should always be really simple so I think we can give it a try!